app/components/search/thumbnail_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if show_thumbnail? %>
+  <%= image_tag thumbnail_url, class: 'thumbnail', alt: '' %>
+<% else %>
+  <svg style="text-anchor: middle" width="240" height="240" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="Placeholder: Responsive image" aria-hidden="true" focusable="false">
```

Comment:
Likewise, the `aria-hidden="true"` here is required for screen readers since this isn't an `img` tag.